### PR TITLE
Emit system information of every run of the CLI

### PR DESCRIPTION
**Pull request by teobugslayer**
Gather system information about the runtime environment.
At install time, warn about potential problems due to mis-configured environment.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,11 +56,11 @@
     "shelljs": "0.3.0",
     "tabtab": "https://github.com/Icenium/node-tabtab/tarball/master",
     "temp": "0.8.1",
-    "unzip": "0.1.9",
+    "unzip": "0.1.11",
```
Comment (rosen-vladimirov) [inline package.json]: Is this change required - do we use unzip anywhere? If not, let's just remove it.

Comment (teobugslayer) [inline package.json]: I do not know, I simply It is require'd in the FS but not actually used

**Comment by rosen-vladimirov** [comment]
:+1:

**Comment by ErjanGavalji** [comment]
:+1:

File: package.json [hunk 2]
```diff
@@ -58,11 +59,10 @@
     "shelljs": "0.3.0",
     "tabtab": "https://github.com/Icenium/node-tabtab/tarball/master",
     "temp": "0.8.1",
-    "unzip": "0.1.9",
+    "winreg": "0.0.12",
+    "xcode": "https://github.com/NativeScript/node-xcode/archive/NativeScript-0.9.tar.gz",
```
Comment (ErjanGavalji) [inline package.json]: why `xcode` but not `node-xcode`?

Comment (teobugslayer) [inline package.json]: This is not my code, I have no idea :)

Comment (rosen-vladimirov) [inline package.json]: because the name of the module in package.json is xcode

Comment (fealebenpae) [inline package.json]: Basically what Rosen said.
